### ISSUE TITLE
Other VMs affecged...

### ISSUE THREAD

**Issue by enigma9o7** [myspaghetti/macos-virtualbox#318]
First, cool script, installed catalina in linux virtual box for me smoothly.

However, somehow it broke three of my existing macOS VMs that were working just fine before.  I tried looking thru the script to see if it's setting something that could affect other things but I don't see it.   I looked through some of the closed issues here hoping sonmebody else had experienced this before and eventually found something that made the one boot I wanted now boot... 
https://github.com/myspaghetti/macos-virtualbox/issues/215

Both bigsur and mojave started doing what's in that screenshot with the UEFI stuff. Both of which I had installed from ISO created on another machine, this was the first time I'd used your script.   Using the suggested command with FS1 worked but I've never ever had to do this before... and hope for some advice to make it permanent...  and Sierra doing something different cuz it destroyed the entry somehow i dont get it so will need to recreate it... i wonder why it only happened to me... or maybe nobody else already had macOS vms?  The only one I have that still worked after using this script to install Catalina is High Sierra.

**Comment by myspaghetti** [comment]
Did you perhaps update VirtualBox to 6.1?

**Comment by enigma9o7** [comment]
I am using Vritualbox 6.1.14 yes...

I did a while ago have 6., but had already upgraded to 6.1 previously
(Mojave didn't work for me with 6.0).

**Comment by enigma9o7** [comment]
Well, I just resolved my issue.  Somehow, some of my other VMs ended up in Catlina folder.  I most certainly did not do this myself. I think this happened because I already had a group named macOS, and had my macOS VMs in it before I started this script.   Then I think t his script used that folder for Catalina.  After the first time I shut down the VM after install, I renamed it in the UI to the longer name, I think it was just macOS at first or something simpler.   

This is what I just saw now; by moving things back where they belong all is resolved :)
  
![shot-2020-10-13_09-59-53](https://user-images.githubusercontent.com/30579733/95892470-76112a00-0d3b-11eb-94d4-d248878f7af8.jpg)
